include/ddc/kernels/splines/spline_builder_2d.hpp

```diff
@@ -87,7 +87,14 @@ class SplineBuilder2D
      */
     using bsplines_type2 = typename builder_type2::bsplines_type;
 
+	/**
+     * @brief The type of the derivatives on boundaries in the first dimension which are compatible with this class.
+     */
     using deriv_type1 = typename builder_type1::deriv_type;
+
+	/**
+     * @brief The type of the derivatives on boundaries in the second dimension which are compatible with this class.
+     */
     using deriv_type2 = typename builder_type2::deriv_type;
```

Comment:
Bad indentation. Beware clang-format doesn't affect comments.

I'm not sure about this description. For me the "type of the derivatives" is usually `double` f'(x) is a real. But I think you are referring to `Deriv` here.